index.js

```diff
@@ -296,4 +307,9 @@ const styles = StyleSheet.create({
     borderColor: '#00FF00',
     backgroundColor: 'transparent',
   },
+<<<<<<< HEAD
+})
+
+=======
 });
+>>>>>>> e7a01fb3f948e5632957970ba453edd5ab977700
```

Comment:
is this from a merge conflict?

Reply:
Oops.  I am sorry for, I just test my code in my brain because I have already leave the App development team, and not yet rebuild my develop environment of React Native